jjb/hiltests/hiltest-sanity.yaml

```diff
@@ -11,8 +11,8 @@
     project-name: hiltest
     archive-artifacts: "**/*.log"
     branch: master
-    build-timeout: 120
-    submodule-timeout: 120
+    build-timeout: 10000
+    submodule-timeout: 10000
     build-node: static-ubuntu-dev-hiltest
     jjb-version: 3.5.0
     build-node-label-check: true
```

Comment:
lets remove this true and the node label list. it will conflict with `build-node`